### PR TITLE
DIABLO-174, login page; messaging for courses-data-table

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/DIABLO-174

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -49,6 +51,7 @@ axios.interceptors.response.use(
 // Vue config
 Vue.config.productionTip = isDebugMode
 Vue.config.errorHandler = function(error, vm, info) {
+  console.error(error || info)
```
Comment (johncrossman) [inline src/main.ts:54]: The client-side error handling works well but we need the raw error logged to console.

File: src/components/course/CoursesDataTable.vue [hunk 2]
```diff
@@ -118,7 +121,10 @@
         <tbody v-if="!items.length">
           <tr>
             <td id="message-when-zero-courses" class="ma-4 text-no-wrap title" :colspan="headers.length">
-              {{ messageWhenZeroCourses }}
+              <span v-if="refreshing">
+
+              </span>
```
Comment (lyttam) [inline src/components/course/CoursesDataTable.vue:126]: @johncrossman Does this empty `span` do anything?

Comment (johncrossman) [reply src/components/course/CoursesDataTable.vue:126]: Whoops. Not a thing. Amending...

**Comment by lyttam** [comment]
Looks good! Thanks.

**Pull request merged**